resources/dicts/relationship_events/normal_interactions/neutral.json

```diff
@@ -157,11 +147,11 @@
 		"id": "neutral_interaction8",
 		"interactions": [
 			"r_c notices m_c sulking around camp again.",
-			"r_c keeps hearing m_c sighing sadly nearby.",

```

Comment:
Removed

Reply:
Didn't really liked how this one flowed <3